pkg/networks/usernet/client.go

```diff
@@ -74,11 +76,19 @@ func (c *Client) ResolveAndForwardSSH(ipAddr string, sshPort int) error {
 }
 
 func (c *Client) ResolveIPAddress(ctx context.Context, vmMacAddr string) (string, error) {
-	timeout := time.After(2 * time.Minute)
+	ResolveIPAddressTimeout := 2 * time.Minute
+	ResolveIPAddressTimeoutEnv := os.Getenv("RESOLVE_IP_ADDRESS_TIMEOUT")
```

Comment:
Is this env var recognized by other programs too?
Otherwise this should have some prefix like `LIMA_USERNET_`

Reply:
How about `LIMA_CUSTOM_TIMEOUT` so that it can be a generic one?

Reply:
I'd it is more confusing, as this var only affects the timeout for resolving IP addresses with usernet.

Reply:
I agree with `LIMA_USERNET_TIMEOUT` to specify the approximate purpose.

> How about LIMA_CUSTOM_TIMEOUT so that it can be a generic one?

The word `CUSTOM` is redundant; every environment variable specifies a custom setting instead of using something hardcoded into the program.

And I think we don't need one "generic" timeout. If we need another timeout override, chances are that we might need a different value.

Reply:
I see you picked `LIMA_USERNET_RESOLVE_IP_ADDRESS_TIMEOUT`, which feels a bit long, but is fine with me.

Reply:
As `LIMA_TIMEOUT_FACTOR` with values `*2` or `+5`

Reply:
`LIMA_TIMEOUT_FACTOR` looks good too, but if we are going to adopt it, we have to make sure that this factor applies to (almost) all the timeout values across the entire code base

Reply:
> LIMA_USERNET_TIMEOUT

`LIMA_USERNET_RESOLVE_IP_ADDRESS_TIMEOUT` looks better, as it is more descriptive.
The length does not really matter, as this env var should be rarely used.

Reply:
I prefer absolute values.

A factor sounds like a good idea, but then you need to know the original value.

And if you want to extend a 10s timeout to 30s, it doesn't mean you want to extend a 5min timeout to 15min. Not all timeouts scale the same way.